connman/plugins/sailfish_ofono.c

```diff
@@ -582,13 +599,102 @@ static int modem_configure(struct modem_data *md)
 	return index;
 }
 
+static gboolean is_modem_network_configured(struct modem_data *md)
```

Comment:
This one I would call `modem_is_network_configured()`

Reply:
True, follows the other code nicely. Changed,